template.html

```diff
@@ -0,0 +1,238 @@
+                margin-left: -365px;
+                top: 50%;
+                margin-top: -265px;
+                border: 2px solid #fff;
+            }
+
+            .pm-certificate-border {
+                position: relative;
+                width: 720px;
+                height: 520px;
+                padding: 0;
+                border: 1px solid #E1E5F0;
+                background-color: rgba(255, 255, 255, 1);
+                background-image: none;
+                left: 50%;
+                margin-left: -360px;
+                top: 50%;
+                margin-top: -260px;
+
+                .pm-certificate-block {
+                    width: 650px;
+                    height: 200px;
+                    position: relative;
+                    left: 50%;
+                    margin-left: -325px;
+                    top: 70px;
+                    margin-top: 0;
+                }
+
+                .pm-certificate-header {
+                    margin-bottom: 10px;
+                }
+
+                .pm-certificate-title {
+                    position: relative;
+                    top: 40px;
+
+                    h2 {
+                        font-size: 34px !important;
+                    }
+                }
+
+                .pm-certificate-body {
+                    padding: 20px;
+
+                    .pm-name-text {
+                        font-size: 20px;
+                    }
+                }
+
+                .pm-earned {
+                    margin: 15px 0 20px;
+
+                    .pm-earned-text {
+                        font-size: 20px;
+                    }
+
+                    .pm-credits-text {
+                        font-size: 15px;
+                    }
+                }
+
+                .pm-certified {
+                    font-size: 12px;
+
+                    .underline {
+                        margin-bottom: 5px;
+                    }
+                }
+
+                .pm-certificate-footer {
+                    width: 650px;
+                    height: 100px;
+                    position: relative;
+                    left: 50%;
+                    margin-left: -325px;
+                    bottom: -105px;
+                }
+            }
+        }
+
+    </style>
+</head>
+<body>
+<div class="container pm-certificate-container">
+    <div class="outer-border"></div>
+    <div class="inner-border"></div>
+
+    <div class="pm-certificate-border col-xs-12">
+        <div class="row pm-certificate-header">
+            <div class="pm-certificate-title cursive col-xs-12 text-center">
+                <h2>Let's Build A... Certificate of Shillfullness</h2>
+            </div>
+        </div>
+
+        <div class="row pm-certificate-body">
+            <div class="pm-certificate-block">
+                <div class="col-xs-12">
+                    <div class="row">
+                        <div class="col-xs-2"><!-- LEAVE EMPTY --></div>
```

Comment:
Who are you leaving these comments for?

Reply:
*I definitely didn't steal this template from somewhere else and strip off some of the unneeded code*

Reply:
_please cite your sources_

Reply:
https://codepen.io/seebrosky/pen/xLzKOV

Reply:
Thank